WMF Framework/WMFAnnouncement.m

```diff
@@ -46,6 +46,12 @@ + (NSValueTransformer *)imageURLJSONTransformer {
         }];
 }
 
++ (MTLValueTransformer *)textJSONTransformer {
+    return [MTLValueTransformer transformerUsingForwardBlock:^id(NSString *text, BOOL *success, NSError *__autoreleasing *error) {
+        return [text stringByReplacingOccurrencesOfString:@"<br>" withString:@"\n"];
```

Comment:
It's a human-edited field so that's possible though unlikely. I think @joewalsh's comment is on-point too...